spec/lib/data_update_scripts/coerce_unicorn_reactions_to_likes_spec.rb

```diff
@@ -0,0 +1,31 @@
+require "rails_helper"
+require Rails.root.join(
+  "lib/data_update_scripts/20220906190412_coerce_unicorn_reactions_to_likes.rb",
+)
+
+describe DataUpdateScripts::CoerceUnicornReactionsToLikes do
+  let(:user) { create(:user) }
+  let(:article) { create(:article) }
+  let(:unicorn_reaction) { create(:reaction, user: user, reactable: article, category: "like") }
+
+  before do
+    unicorn_reaction.update_attribute(:category, "unicorn") # This skips validations
```

Comment:
We have to play tricks to get a legacy unicorn reaction in the database since we've already removed it from the list of acceptable reactions 😉